Formula/navi.rb

```diff
@@ -11,14 +11,18 @@ class Navi < Formula
     sha256 "64b6dac7d4c689ab6aa5c88ef58369249af2d9a6ac44f56769685569a673dd56" => :high_sierra
   end
 
+  depends_on "rust" => :build
   depends_on "fzf"
 
   def install
-    libexec.install Dir["*"]
-    bin.write_exec_script(libexec/"navi")
+    libexec.install Dir["cheats/*"]
+    libexec.install Dir["shell/*"]
+    system "cargo", "install", "--root", prefix, "--path", "."
+    mv "#{bin}/navi", "#{libexec}/navi"
```

Comment:
I dont know why we cannot install the binary directly from the cargo output. Can you elaborate little bit in here?

Reply:
Yes, I had added a comment but I was asked to remove it haha

The problem is that `navi` reads files from `${current_path}/cheats`, where `$current_path` is the path of the executable binary itself. 

The binary must be in the same folder as the folder `/cheats` then.

That's why I move it to `libexec` and create a symlink.

Reply:
That seems like a bug that should be reported upstream.

Reply:
I've changed in the code to check for `${current_path}/cheats` and `${current_path}/../libexec/cheats` as workaround.